lib/matplotlib/colorbar.py

```diff
@@ -644,8 +634,7 @@ def update_ticks(self):
         locator, formatter = self._get_ticker_locator_formatter()
         long_axis = ax.yaxis if self.orientation == 'vertical' else ax.xaxis
         if self._use_auto_colorbar_locator():
-            _log.debug('Using auto colorbar locator on colorbar')
-            _log.debug('locator: %r', locator)
+            _log.debug('Using auto colorbar locator %r on colorbar', locator)
```

Comment:
I put this in as a debugging step at some point I think.  Not sure what our policy is vis-a-vis lots of debugs, so feel free to remove it if its too crufty.

Reply:
I think leaving one log line is fine.